### PR TITLE
NO_MERGE Feature/multi_load PATCH

### PR DESCRIPTION
**Pull request by ViljoenFritz**
CHANGELOG entry:

### Added
- [Add load statuses for multi loads](https://github.com/NoSoft-SA/nspack/pull/452/commits/c246fc1aa8abeb54da2141155fb5a578ce5f4de5)
### Changed
- [Remove Close button from loads](https://github.com/NoSoft-SA/nspack/pull/452/commits/b680ece183a27bdb49826df13356ad4b4b3011d0)
- [Split multi loads between shipped and unshipped](https://github.com/NoSoft-SA/nspack/pull/452/commits/598a3b7b43dada0ca8fda3aac79778efb5c6b8d2)
### Fixed
- [Fix response messages for loads](https://github.com/NoSoft-SA/nspack/pull/452/commits/5106429e8d8203ec1afbfa13f2398bcc0be0c558)

### CODE REVIEW

**Review by jcsjcs — CHANGES_REQUESTED** [review]

File: grid_definitions/lists/loads_multi.yml [hunk 1]
```diff
@@ -14,14 +14,25 @@
...
   :hide_if_true: shipped
 - :separator: true
-# :page_controls:
-# - :control_type: :link
-#   :url: "/list/loads_multi_new/multi?key=standard"
-#   :text: New Multi Load
-#   :style: :button
-#   :behaviour: :popup
+:page_controls:
+- :control_type: :link
```
[the remark this answers is not in the document]
Comment (ViljoenFritz) [reply grid_definitions/lists/loads_multi.yml:18]: I dont like Metrics/CyclomaticComplexity

[the remark this answers is not in the document]
Comment (jcsjcs) [reply grid_definitions/lists/loads_multi.yml:18]: ??? in a YAML file?
Why does the page that is all about grouping existing loads have a button to create a new load?

[the remark this answers is not in the document]
Comment (ViljoenFritz) [reply grid_definitions/lists/loads_multi.yml:18]: this comment was meant for the back button

[the remark this answers is not in the document]
Comment (ViljoenFritz) [reply grid_definitions/lists/loads_multi.yml:18]: ill remove the new load button

**Pull request merged**